lib/metrifox_sdk/usages/api.rb

```diff
@@ -22,7 +22,7 @@ def fetch_access(base_url, api_key, request_payload)
       parse_response(response, "Failed to check access")
     end
 
-    def fetch_usage(base_url, api_key, request_payload)
+    def record_usage(base_url, api_key, request_payload)
```

Comment:
Why’s fetch changing to record? It’s the same url but one is get and the other post. They’re both retained for fetching events and recording.